components/media_player/ui/Card.qml

```diff
@@ -205,119 +285,221 @@ Rectangle {
+
+                    onClicked: {
+                        Haptic.playEffect(Haptic.Click);
+                        cardSwipeView.setCurrentIndex(features.indexOf("MEDIAPLAYERREMOTE"));
+                    }
                 }
             }
-        }
 
-        // speakers
-        Text {
-            visible: obj.isSupported(MediaPlayer.F_SPEAKER_CONTROL) ? true : false
-            color: Style.color.text
-            opacity: cardSwipeView.currentIndex === features.indexOf("SPEAKER_CONTROL") ? 1 : 0.5
-            text: Style.icon.speaker
-            renderType: Text.NativeRendering
-            width: 60; height: 60
-            verticalAlignment: Text.AlignVCenter; horizontalAlignment: Text.AlignHCenter
-            font {family: "icons"; pixelSize: 80 }
-
-            MouseArea {
-                anchors.fill: parent
-                width: parent.width + 30; height: width
+            //EPGView
+            Text {
+                visible: obj.isSupported(MediaPlayer.F_MEDIAPLAYEREPGVIEW) ? true : false
+                color: Style.color.text
+                opacity: cardSwipeView.currentIndex === features.indexOf("MEDIAPLAYEREPGVIEW") ? 1 : 0.5
+                text: Style.icon.remote
+                renderType: Text.NativeRendering
+                width: 60; height: 60
+                verticalAlignment: Text.AlignVCenter; horizontalAlignment: Text.AlignHCenter
+                font {family: "icons"; pixelSize: 80 }
+
+                MouseArea {
+                    anchors.fill: parent
+                    width: parent.width + 30; height: width
+
+                    onClicked: {
+                        Haptic.playEffect(Haptic.Click);
+                        cardSwipeView.setCurrentIndex(features.indexOf("MEDIAPLAYEREPGVIEW"));
+                    }
+                }
+            }
 
-                onClicked: {
-                    Haptic.playEffect(Haptic.Click);
-                    cardSwipeView.setCurrentIndex(features.indexOf("SPEAKER_CONTROL"));
+            // speakers
+            Text {
+                visible: obj.isSupported(MediaPlayer.F_SPEAKER_CONTROL) ? true : false
+                color: Style.color.text
+                opacity: cardSwipeView.currentIndex === features.indexOf("SPEAKER_CONTROL") ? 1 : 0.5
+                text: Style.icon.speaker
+                renderType: Text.NativeRendering
+                width: 60; height: 60
+                verticalAlignment: Text.AlignVCenter; horizontalAlignment: Text.AlignHCenter
+                font {family: "icons"; pixelSize: 80 }
+
+                MouseArea {
+                    anchors.fill: parent
+                    width: parent.width + 30; height: width
+
+                    onClicked: {
+                        Haptic.playEffect(Haptic.Click);
+                        cardSwipeView.setCurrentIndex(features.indexOf("SPEAKER_CONTROL"));
+                    }
                 }
             }
-        }
-    }
+
+        }}
```

Comment:
I know it's nitpicking, but can we break these into two lines and indent them correctly? :) Thanks